python/xoscar/constants.py

```diff
@@ -19,3 +19,5 @@
 
 # unix socket.
 XOSCAR_UNIX_SOCKET_DIR = XOSCAR_TEMP_DIR / "socket"
+
+XOSCAR_CONNECT_TIMEOUT = 8
```

Comment:
This might be a break change which could cause unexpected behavior, can you get the value from environment variable and I suggest to keep the behavior compatible.